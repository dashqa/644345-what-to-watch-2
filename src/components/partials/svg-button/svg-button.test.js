import React from 'react';
import renderer from 'react-test-renderer';
import SvgButton from './svg-button';

const props = {
  classes: `play`,
  title: `Play button`,
  svgViewBox: `0 0 19 19`,
  svgWidth: `19`,
  svgHeight: `19`,
  svgXlink: `#play-s`,
  onClick: () => {}
};

it(`Svg button correctly renders after relaunch`, () => {
  const tree = renderer.create(<SvgButton {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
