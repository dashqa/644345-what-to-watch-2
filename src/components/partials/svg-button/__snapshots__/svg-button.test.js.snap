// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg button correctly renders after relaunch 1`] = `
<button
  className=""
  data-icon=""
  onClick={[Function]}
  type="button"
>
  <svg
    height=""
    viewBox=""
    width=""
  >
    <use
      xlinkHref=""
    />
  </svg>
  <span>
    
  </span>
</button>
`;
